src/app/drive/components/FileViewer/utils/fileViewerWrapperUtils.ts

```diff
@@ -0,0 +1,220 @@
+import {
+  contextMenuDriveItemShared,
+  contextMenuDriveItemSharedAFS,
+  contextMenuDriveNotSharedLink,
+  contextMenuTrashItems,
+} from '../../DriveExplorer/DriveExplorerList/DriveItemContextMenu';
+import { ListItemMenu } from 'app/shared/components/List/ListItem';
+import { AdvancedSharedItem, PreviewFileItem } from 'app/share/types';
+import { DriveItemData } from 'app/drive/types';
+import { getAppConfig } from 'app/core/services/config.service';
+import {
+  canFileBeCached,
+  getDatabaseFileSourceData,
+  updateDatabaseFileSourceData,
+} from 'app/drive/services/database.service';
+import dateService from 'app/core/services/date.service';
+import { DriveItemActions } from '../../DriveExplorer/DriveExplorerItem/hooks/useDriveItemActions';
+import { UserSettings } from '@internxt/sdk/dist/shared/types/userSettings';
+
+export type TopBarActionsMenu = ListItemMenu<DriveItemData> | ListItemMenu<AdvancedSharedItem>;
+
+type PathProps = 'drive' | 'trash' | 'shared' | 'recents';
+
+interface FileViewerShortcuts {
+  renameItemFromKeyboard: ((item) => void) | undefined;
+  removeItemFromKeyboard: ((item) => void) | undefined;
+}
+
+export const TopDropdownBarActionsMenu = ({
```

Comment:
This is not a component if I am not mistaken, by convention the functions use `camelCase` and components `PascalCase`